### PR TITLE
Updated ip for use with rpi3b

### PR DESCRIPTION
**Pull request by snazzybunny**
Instructions on how to install are found in README.md. It would be a good idea to find a way to include the npm plugins like commander and underscore as well as nodejs v8 into the image, that is, if we were to use this program. Otherwise, you would need Internet before being able to use this program.

Tested on rpi3:
default, detectrpi, swifi, wifi, rename

Needs work on rpi3:
ethernet, expandfs

### CODE REVIEW

**Comment by snazzybunny** [comment]
If we can install node and its dependencies on treeimage builder that would be good. I assume we can include the npm plugins with the ip program so the user doesnt need Internet to download them.

File: README.md [hunk 1]
```diff
@@ -9,12 +7,33 @@
     rename <hostname>                                         changes Hostname
     adapter <wirelessSSID> <password> <wirelessSecurityType>  [deprecated] connects a adapter to a wifi network
     wifi <ESSID> [password]                                   connects to a wifi network
-    ethernet <ip> <mask> <gateway> <dns>                      configures rpi network interface to a static ip address
+    swifi <ip> <mask> <gateway> <dns>                      configures rpi wifi interface to a static ip address
+
+    sethernet <ip> <mask> <gateway> <dns>                      configures rpi ethernet interface to a static ip address
```
Comment (dogi) [inline README.md]: @snazzybunny please do not remove old subcommands

File: lib/DetectRPI.js [hunk 2]
```diff
@@ -23,7 +23,8 @@ module.exports = function() {
     "0010" : "RPIB+",
     "0011" : "CM",
     "a11041" : "RPI2B",
-    "a21041" : "RPI2B"
+    "a21041" : "RPI2B",
+    "a02082" : "RPI3B"
```
Comment (dogi) [inline lib/DetectRPI.js]: @snazzybunny we should also add comments for raspberry pi zero and pi zero W, so that other maybe fill this gaps ...

**Review by dogi — CHANGES_REQUESTED** [review]
please change following

**Review by dogi — APPROVED** [review]

**Pull request merged**